src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1359,7 +1363,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
 
             // We want to report diagnostics with source suppression in the error log file.
             // However, these diagnostics won't be reported on the command line.
-            var reportSuppressedDiagnostics = errorLogPath != null;
+            var reportSuppressedDiagnostics = errorLogOptions?.Path != null;
```

Comment:
I agree that it doesn't make sense for there to be an `ErrorLogOptions` object without a `Path` property, and in fact the helper method `CommandLineParser.ParseErrorLogOptions` ensures that this doesn't happen. So ok, I changed the test to `errorLogOptions != null`.

---
In reply to: [322389703](https://github.com/dotnet/roslyn/pull/37148#discussion_r322389703) [](ancestors = 322389703)

Reply:
Lately, I prefer the `errorLogOptions is null` or `errorLogOptions is object` to test if a reference type value is `null`. Just because I can create havoc by defining equality and inequality operators that return true when comparing with `null`.

Reply:
I love it. Changed to `errorLogOptions is object`, and I'm going to use that myself from now on.

---
In reply to: [323097707](https://github.com/dotnet/roslyn/pull/37148#discussion_r323097707) [](ancestors = 323097707)

Reply:
One more, @jaredpar. 😄